File: 09. Array/D. Rotasi Matriks/d_rotasi_matriks.java
import java.util.Scanner;

class d_rotasi_matriks {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int N = scan.nextInt();
        int M = scan.nextInt();
        int[][] arr = new int[N][M];
        for (int i = 0; i < N; i++) {
            for (int j = 0; j < M; j++) {
                arr[i][j] = scan.nextInt();
            }
        }
        scan.close();
        for (int i = 0; i < M; i++) {
            for (int j = (N - 1); j >= 0; j--) {
                System.out.print(arr[j][i] + " ");
            }
            System.out.println();
        }
    }
}

File: 09. Array/D. Rotasi Matriks/d_rotasi_matriks.go
package main

import (
  "fmt"
)

func main() {
  var N, M int
  fmt.Scanf("%d %d", &N, &M)
  var arr [101][101]int
  for i := 0; i < N; i++ {
    for j := 0; j < M; j++ {
      if j == M-1 {
        fmt.Scanln(&arr[i][j])
      } else {
        fmt.Scan(&arr[i][j])
      }
    }
  }
  for i := 0; i < M; i++ {
    for j := (N - 1); j >= 0; j-- {
      fmt.Print(arr[j][i], " ")
    }
    fmt.Println()
  }
}
